sources/notion/helpers/client.py

```diff
@@ -151,3 +151,14 @@ def search(
             next_cursor = response.get("next_cursor")
             has_more = next_cursor is not None
             start_cursor = next_cursor
+
+    def get_database(self, database_id: str) -> Dict[str, Any]:
```

Comment:
linter failed. change Dict[str, Any] to Any.

Reply:
the rest looks good, well done))

Reply:
don't forget to edit docstring

Reply:
Changes done. Please see.

Reply:
PR passed the checks @AstrakhantsevaAA 👍  please review.  Thanks